bindata/etcd/pod.yaml

```diff
@@ -10,6 +10,35 @@ metadata:
     revision: "REVISION"
 spec:
   initContainers:
+    - name: etcd-ensure-env-vars
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      command:
+        - /bin/sh
+        - -c
+        - |
+          #!/bin/sh
+          set -euo pipefail
+
+          if [ -z "$NODE_NODE_ENVVAR_NAME_ETCD_URL_HOST"]; then
```

Comment:
these are still unbound right?

These blocks can be simplified to something like below 

```
: "${NODE_NODE_ENVVAR_NAME_ETCD_NAME?not set}"
```
ref: https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion

Reply:
@alaypatel07 if you tested this you would have seen other errors as well, you need a space after quote and before right bracket.
... `URL_HOST"]; then`
```
 [: missing `]'
```